massa-proof-of-stake-exports/Cargo.toml

```
@@ -28,3 +28,4 @@ massa_serialization = { path = "../massa-serialization" }
 # for more information on what are the following features used for, see the cargo.toml at workspace level
 [features]
 instrument = ["tokio/tracing", "massa_models/instrument"]
+sandbox = []
```

Need to propagate here also.
```suggestion
sandbox = ["massa_models/sandbox"]
```

---

done